Gopkg.toml

```diff
@@ -49,7 +49,7 @@ required = [
   name = "github.com/giantswarm/apiextensions"
 
 [[constraint]]
-  branch = "master"
+  branch = "app-support"
```

Comment:
Changes LGTM in general but I don't see where the new method is used. What am I missing?

Reply:
It's because before there was a service in `service/controller/app/v1/kubeconfig/kubeconfig.go` that used the key functions to get the kubeconfig secret name and namespace.

It gets deleted and the was code moved to the kubeconfig library in https://github.com/giantswarm/kubeconfig/pull/3.

Reply:
Now I get it 👍